src/main/java/net/coderbot/iris/uniforms/CommonUniforms.java

```diff
@@ -46,14 +49,25 @@ public static void addCommonUniforms(UniformHolder uniforms, IdMap idMap) {
 			.uniform1i(ONCE, "lightmap", TextureUnit.LIGHTMAP::getSamplerId)
 			.uniform1b(PER_FRAME, "hideGUI", () -> client.options.hudHidden)
 			.uniform1i(ONCE, "noisetex", () -> 15)
-			.uniform1f(PER_FRAME, "eyeAltitude", () -> Objects.requireNonNull(client.getCameraEntity()).getY())
+			.uniform1f(PER_FRAME, "eyeAltitude", () -> Objects.requireNonNull(client.getCameraEntity()).getEyeY())
 			.uniform1i(PER_FRAME, "isEyeInWater", CommonUniforms::isEyeInWater)
 			.uniform1f(PER_FRAME, "blindness", CommonUniforms::getBlindness)
 			.uniform1i(PER_FRAME, "heldBlockLightValue", new HeldItemLightingSupplier(Hand.MAIN_HAND))
 			.uniform1i(PER_FRAME, "heldBlockLightValue2", new HeldItemLightingSupplier(Hand.OFF_HAND))
 			.uniform1f(PER_FRAME, "nightVision", CommonUniforms::getNightVision)
 			.uniform1f(PER_FRAME, "screenBrightness", () -> client.options.gamma)
-			.uniform1f(PER_TICK, "playerMood", CommonUniforms::getPlayerMood);;
+			.uniform1f(PER_TICK, "playerMood", CommonUniforms::getPlayerMood)
+			.uniform2i(PER_FRAME, "eyeBrightness", CommonUniforms::getEyeBrightness)
+			.uniform1f(PER_TICK, "rainStrength", CommonUniforms::getRainStrength)
```

Comment:
Should be per frame since this is smoothed between ticks (that's what tickDelta is for)